packages/components/src/color-palette/test/__snapshots__/index.js.snap

```diff
@@ -357,6 +357,11 @@ exports[`ColorPalette Dropdown should render it correctly 1`] = `
 
 <Dropdown
   contentClassName="components-color-palette__custom-color-dropdown-content"
+  popoverProps={
```

Comment:
Just noting that this snapshot change is due to a change in `CustomColorPickerDropdown` (in `packages/components/src/color-palette/index.js`), where previously the `__unstableShift` prop was set only when `isRenderedInSidebar` was `true` — while, with the changes in this PR, `__unstableShift` is _always_ set to `true`